packages/adblocker-benchmarks/blockers/brave.js

```diff
@@ -6,19 +6,21 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-const { Engine } = require('adblock-rs');
+const { Engine, FilterSet } = require('adblock-rs');
 
 module.exports = class Brave {
   static parse(rawLists) {
-    return new Brave(new Engine(rawLists.split(/[\n\r]+/g)));
+    const filterSet = new FilterSet(false);
```

Comment:
The `false` and `true` here are for debug information and optimization respectively, which is how I imagine it would be used in production.

The [`from_rules`](https://github.com/brave/adblock-rust/blob/9a2b6d252c38836bfcfadefa9a07b1848855305f/src/engine.rs#L39) function in the Rust implementation also uses these values.